node-wasm/js/README.md

```diff
@@ -0,0 +1,31 @@
+# Lumina node wasm
+
+A compatibility layer for the [`Lumina`](https://github.com/eigerco/lumina) node to
+work within a browser environment and be operable with javascript.
+
+# Example
+Starting lumina inside a dedicated worker
+
+```javascript
+import { spawnNode, NodeConfig, Network } from "lumina-node";
+
+const node = await spawnNode();
+const mainnetConfig = NodeConfig.default(Network.Mainnet);
+
+await node.start(mainnetConfig);
+
+await node.wait_connected();
+await node.request_head_header();
+```
+
+Note that `spawnNode` implicitly calls wasm initialisation code. If you want to set things up manually, make sure to call the default export before using any of the wasm functionality.
+
+```javascript
+import init, { NodeConfig, Network } from "lumina-node";
+
+await init();
+const config = NodeConfig.default(Network.Mainnet);
+
+console.log(config.bootnodes);
+
```

Comment:
can you add also how to create the Node etc?

Reply:
do you mean the worker?